README.md

```diff
@@ -52,8 +52,7 @@ Supported VFX Platforms
 -----------------
 The current release supports:
 - VFX platform 2021, 2020, 2019, 2018
-- Python 2.7 - 3.9
-- Notice that Python 2.7 is deprecated & we plan to drop it in OTIO release 0.15
```

Comment:
Plans kind of changed, so I thought we should remove this until we have a clear timeline on when 2.7 support will be dropped before adding a deprecation notice.

Reply:
We do have a clear timeline ~ as soon as CY2023 is specified, CY2019 will drop off and with it, py27.